core/modules/url_filter.h

```diff
@@ -21,6 +24,9 @@ class Flow {
   uint32_t dst_ip;
   uint16_t src_port;
   uint16_t dst_port;
+  uint32_t padding;
+
+  Flow() : padding(0) {}
```

Comment:
I think this example shows the case we're dealing with here, which would mean the other values are not getting zeroed:

http://en.cppreference.com/w/cpp/language/default_initialization


`    const T2 t2;      // const class, calls the user-provided default ctor
                      // t2.mem is default-initialized (to indeterminate value)`